crates/sdl-encoder/src/union_def.rs

```diff
@@ -0,0 +1,106 @@
+use std::fmt::{self, Display};
+
+/// UnionDefs are an abstract type where no common fields are declared.
+///
+/// *UnionDefTypeDefinition*:
+///     Description<sub>opt</sub> **union** Name Directives<sub>\[Const\] opt</sub> UnionDefMemberTypes<sub>opt</sub>
+///
+/// Detailed documentation can be found in [GraphQL spec](https://spec.graphql.org/draft/#sec-UnionDef).
+///
+/// ### Example
+/// ```rust
+/// use sdl_encoder::{UnionDef};
+///
+/// let mut union_ = UnionDef::new("Cat".to_string());
+/// union_.description(Some(
+///     "A union of all cats represented within a household.".to_string(),
+/// ));
+/// union_.member("NORI".to_string());
+/// union_.member("CHASHU".to_string());
+///
+/// assert_eq!(
+///     union_.to_string(),
+///     r#""""A union of all cats represented within a household."""
+/// union Cat = NORI | CHASHU
```

Comment:
Note that unions do not contain values as members, but always refer to object types.

Reply:
I do not understand this comment. Do you mean the naming is incorrect? The [spec seems to call these values as `members`](https://spec.graphql.org/draft/#sec-Unions)